core/package-graph/index.js

```diff
@@ -130,6 +130,38 @@ class PackageGraph extends Map {
     return result.map(node => node.pkg);
   }
 
+  /**
+   * Takes a list of Packages and returns a list of those same Packages with any Packages
+   * that depend on them. i.e if packageC depended on packageD `graph.addDependents([packageD])`
+   * would return [packageD, packageC].
+   *
+   * @param {!Array.<Package>} filteredPackages The packages to include dependents for.
+   * @return {Array.<Package>} The packages with any dependents that weren't already included.
+   */
+  addDependents(filteredPackages) {
```

Comment:
@evocateur agreed!

I debated adding a `breadthFirstSearch` method that takes a `childrenFunc`, which can be used for either `localDependencies` or `localDependents` - what do you think of that? It feels a little odd to expose that as a method, but I couldn't make it a separate helper function without pulling out the call to `this.get` somehow.

Reply:
> It feels a little odd to expose [`breadthFirstSearch`] as a method

There's precedent in this class with the `prune` method calling `remove`, so I think that's fine. It might even prove to be more useful in the future!

```js
  addDependencies(filteredPackages) {
    return this.extendList(filteredPackages, "localDependencies");
  }

  addDependents(filteredPackages) {
    return this.extendList(filteredPackages, "localDependents");
  }

  extendList(packageList, nodeProp) {
    // shared code
  }
```

`extendList` struck me as a little more descriptive than `breadthFirstSearch`

Reply:
Thanks! I did the refactor and added a test. Let me know if there's anything else you'd like to see